### DISCUSSION TITLE
S command in visual line or cs'" in normal mode don't work - help setup

### DISCUSSION THREAD

**Discussion by victorheid** [kylechui/nvim-surround#270] [Q&A] [answered]
Some of the commands shown in the readme seem to conflict with default vim commands (s/S deletes char/line and put in insert mode) and c is delete...

Am I missing something?

**Comment by kylechui** [comment, marked as answer]
The main thing to notice is that the commands are valid for their respective modes. `S` is not normally mapped in visual mode, and the keybind for change surround is `cs`, not just `c`. Since `s` is not a valid motion, people do not normally type `cs` and expect it to do anything; hence we use that for "change surround" instead. All else aside, I encourage you to try the plugin out (or `vim-surround`) and see how the plugin keymaps work for yourself!

**Comment by victorheid** [comment]
I feel stupid now, the issue was my macOS keyboard language which makes
quotes “hang” and not be sent as normal keystrokes (for cs’”) and for
visual mode for some reason I was not typing capital S

Apologies and thanks

**Reply by kylechui** [reply]
No worries, glad your issue got resolved!
